mutual_debt/main.py

```diff
@@ -0,0 +1,51 @@
+"""Mutual Debt Simplification
+
+Usage:
+  simplify-debts <data_file>
+  simplify-debts (-h | --help)
+
+Options:
+  -h --help     Show this screen.
+"""
+import json
+import sys
+
+from mutual_debt.mutual_debt_simplification import debt_list_to_graph, \
+    simplify_debt_graph, draw_graph
+
+
+def print_error(*args, sep=' ', end='\n'):
+    """ Prints values to the stderr stream """
+    print("ERROR:", *args, sep, end, file=sys.stderr)
+
+
+def main():
+
+    if len(sys.argv) != 2:
+        print(__doc__)
```

Comment:
Nice.

Reply:
I definitely agree with this one :smile: